{"name": "创建英雄失败的测试用例-(攻+1)", "status": "passed", "parameters": [{"name": "success_for_name", "value": "'123@qq.com'"}, {"name": "success_for_volume", "value": "1"}, {"name": "fail_for_power", "value": "0"}], "start": 1684150564812, "stop": 1684150564812, "uuid": "08d2a34c-d8ab-4e84-b733-8d74eedf2bb9", "historyId": "67f03cebe876b4b3cb9040c44c9e3a10", "testCaseId": "e6505b56913fd0a5b5c514cb9e6b3fa7", "fullName": "test_create_hero_homework_01.TestHeroCreate#test_create_hero_fail_for_power_p1", "labels": [{"name": "tag", "value": "run(order=7)"}, {"name": "suite", "value": "test_create_hero_homework_01"}, {"name": "subSuite", "value": "TestHeroCreate"}, {"name": "host", "value": "LAPTOP-0EHG7NSH"}, {"name": "thread", "value": "48660-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_create_hero_homework_01"}]}